playgrounds/angular/src/global.css

```
@@ -0,0 +1,26 @@
+@import 'tailwindcss';
+
+/* Lexend 400 */
+@font-face {
+  font-family: 'Lexend';
```

Stylelint (17.12.0)

[error] Expected no quotes around "Lexend" (font-family-name-quotes)

(font-family-name-quotes)